python-virtual-environment-prompt.zsh

```diff
@@ -0,0 +1,4 @@
+export VIRTUAL_ENV_DISABLE_PROMPT=1
+function python_venv_info() {
+  [[ -n "$VIRTUAL_ENV" ]] && echo "(`basename $VIRTUAL_ENV`) "
```

Comment:
I'd prefer to solve this without an additional subshell. For example https://github.com/woefe/git-prompt.zsh/blob/f1681cabe5a42f3b53aa11d691ab2775d2d9b7f4/examples/pure.zsh#L45